packages/material/src/util/layout.tsx

```diff
@@ -72,7 +72,7 @@ export const MaterialLayoutRenderer = ({
         <Grid
           container
           direction={direction}
-          spacing={direction === 'row' ? 16 : 0}
+          spacing={direction === 'row' ? 10 : 0}
```

Comment:
I fear you missed this one.

Reply:
Very spooky 👻

Have updated!

Reply:
Great, my German to English translation failed me ;-)

Reply:
Haha 😂 not at all! Apologies I was making a bad joke.